src/scanner.ts

```diff
@@ -120,13 +133,25 @@ export async function doExport() {
   return outputStream.toString()
 }
 
-export function doZero() {}
+export async function doZero() {
+  await init(true)
+  fsExtra.emptyDirSync(ballotImagesPath)
+  fsExtra.emptyDirSync(scannedBallotImagesPath)
+}
 
 export async function getStatus() {
-  return { batches: await batchStatus() }
+  const batches = await batchStatus()
+  if (election) {
+    return { electionHash: 'hashgoeshere', batches: batches }
+  } else {
+    return { batches: batches }
+  }
 }
 
-export function shutdown() {
+export async function shutdown() {
```

Comment:
Can we not introduce a new term (`shutdown`) when we have a existing term already? 😄 

```suggestion
export async function unconfigure() {
```